pkg/driver/mock/auto/FluidFlow.go

```diff
@@ -0,0 +1,41 @@
+package auto
+
+import (
+	"context"
+	"math/rand/v2"
+	"time"
+
+	"github.com/vanti-dev/sc-bos/pkg/gen"
+	"github.com/vanti-dev/sc-bos/pkg/gentrait/fluidflowpb"
+	"github.com/vanti-dev/sc-bos/pkg/task/service"
+)
+
+func FluidFlow(model *fluidflowpb.Model) service.Lifecycle {
+	s := service.New(service.MonoApply(func(ctx context.Context, _ string) error {
+		go func() {
+			timer := time.NewTimer(durationBetween(30*time.Second, 2*time.Minute))
+			for {
+				direction := gen.FluidFlow_Direction(rand.IntN(int(gen.FluidFlow_BLOCKING) + 1))
+				state := &gen.FluidFlow{
+					FlowRate:             ptr(float32Between(1, 100)),
+					DriveFrequency:       ptr(float32Between(0, 100)),
+					TargetFlowRate:       ptr(float32Between(1, 100)),
+					TargetDriveFrequency: ptr(float32Between(0, 100)),
+					Direction:            direction,
+				}
```

Comment:
I think this simulation will be unrealistic. Notably when the flow is blocked you will still have a flow rate and drive frequency, which is not something I'd expect to see in the real world.

I also think the direction sim would be easier to read as `direction := oneOf(FLOW, RETURN, BLOCKING)`